content/en/docs/reference/glossary/security-context.md

```diff
@@ -4,14 +4,22 @@ id: security-context
 date: 2018-04-12
 full_link: /docs/tasks/configure-pod-container/security-context/
 short_description: >
-  The securityContext field defines privilege and access control settings for a Pod or Container, including the runtime UID and GID.
+  The securityContext field defines privilege and access control settings for a Pod or container, including the runtime UID and GID.
 
 aka: 
 tags:
 - security
 ---
- The securityContext field defines privilege and access control settings for a Pod or Container, including the runtime UID and GID.
+ The `securityContext` field defines privilege and access control settings for
+a {{< glossary_tooltip text="Pod" term_id="pod" >}} or
+{{< glossary_tooltip text="container" term_id="container" >}},
+including the runtime UID and GID.
 
-<!--more--> 
+<!--more-->
 
-The securityContext field in a {{< glossary_tooltip term_id="pod" >}} (applying to all containers) or container is used to set the user, groups, capabilities, privilege settings, and security policies (SELinux/AppArmor/Seccomp) and more that container processes use.
+The `securityContext` field in a Pod or container is used to set the user,
+groups, capabilities, privilege settings, and security policies
+(SELinux/AppArmor/Seccomp) and more that container processes use.
```

Comment:
nit: 
and more [what?]
possible reword:
that container processes use.

Reply:
@kbhawkey this pull request aims to add cross-references; I'm leaving the original wording where I can.

Reply:
Saw your comment. Can you clean up line 22 (remove more). This sentence is hard to read.

Reply:
In the existing text, “and more” means that:
- user
- groups
- capabilities
- privilege settings
- security policies
is not an exhaustive list.

It _is_ hard to read.

Reply:
Reworded; hope that's better.